server/webhook.go

```diff
@@ -548,6 +556,14 @@ func getFileInfo(fileItem vcs.DistinctFileItem, repositoryList []*api.Repository
 	var fType fileType
 	var fileRepositoryList []*api.Repository
 	for _, repository := range repositoryList {
+		if repository.Project.RowStatus == api.Archived {
```

Comment:
Instead of doing the filtering here, let’s add a status filter for RepositoryFind() instead.

Reply:
How about just updating the repo filter func for the VCS SQL review endpoint? https://github.com/bytebase/bytebase/pull/3001/files#diff-e3302fa7fe65284d10a8f46bad75c6e182a9b39a0400fb09f888d0dde5a79bf9R171